FireboltNETSDK/FireboltClient.cs

```diff
@@ -244,36 +248,37 @@ public async Task<GetSystemEngineUrlResponse> GetSystemEngineUrl(string accountN
     private async Task<T> SendAsync<T>(HttpMethod method, string uri, string? body, bool requiresAuth,
         CancellationToken cancellationToken)
     {
-        return await SendAsync<T>(method, uri, body, "application/json", requiresAuth, cancellationToken);
```

Comment:
Same as before with the unnamed true parameter. Please look through the PR to identify other cases.